docs/plugins/gmusic.rst

```diff
@@ -0,0 +1,38 @@
+Gmusic Plugin
+=============
+
+This plugin allows you to manage your Google Play Music library with beets.
```

Comment:
The docs should probably explain that you need to install the `gmusicapi` module from pip to use this plugin, right? (See other plugin documentation for examples—e.g., `lyrics` or `lastgenre`.)